docs/guides/search-path.md

```diff
@@ -124,14 +129,13 @@ steampipe check benchmark.cis_v140 --search_path_prefix aws_prod
 
 ## Tips & Tricks
 
-- Manage your "default" search path with a good connection naming strategy. For most users, this means aggregator first.  For example, I tend to use the plugin name as the name of my aggregator (e.g. `aws`), and as a prefix to the other connections (e.g. `aws_prod`, `aws_dev`, etc).  By doing this, my aggregator is always first, even when adding and removing connections.
-- Use the search path **prefix** command or argument when modifying the search path when you just want to prefer a connection.  
+- Manage your default search path with a good connection-naming strategy. For most users, this means aggregator first.  With AWS, for example, use the plugin name as the name of the aggregator (e.g. `aws`), and as a prefix to the other connections (e.g. `aws_prod`, `aws_dev`, etc).  With this approach the aggregator always comes first, even when adding and removing connections.
+- Use the search path **prefix** command or argument to modify the search path when you want to prefer a connection.  
 - When writing mods, use **unqualified** table names:
   - Qualified names would require you to know the connection names, which you don't know (they are defined by the user).
-  - Users of your mod can use the search path to target different connections
+  - Users of your mod can vary the search path to target different connections
 - If you create custom views or other objects, make sure you keep the `public` schema in your path.
-- Since the `public` schema is first (by default), you can create your own tables and views to use instead of the steampipe tables.  If, for example, there is a table that you want to 'permanently' cache (or only manually refresh), you can create a materialized view with the same name: `create materialized view aws_iam_credential_report as select * from aws_iam_credential_report`.   
-
+- Since the `public` schema is first (by default), you can create your own tables and views to use instead of the Steampipe tables.  If, for example, there is a table that you want to permanently cache, you can create a table with the same name: `create table  aws_iam_credential_report as select * from aws_iam_credential_report`. If you want to be able to refresh that table periodically, you can use a materialized view instead -- `create materialized view aws_iam_credential_report as select * from aws_iam_credential_report` -- and then periodically use `refresh materialized view aws_iam_credential_report`.
```

Comment:
original is better - a table is not a 'cache' but a copy - its not connected to the original source.